web/modules/custom/ingredient_term/ingredient_term.module

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/**
+ * @file
+ * Contains hook for Ingredient Term module.
+ */
+
+declare(strict_types = 1);
+
+use Drupal\Core\Entity\EntityInterface;
+use Drupal\taxonomy\Entity\Term;
+
+/**
+ * Implements hook_entity_presave().
+ */
+function ingredient_term_entity_presave(EntityInterface $entity): void {
+  if ($entity->getEntityTypeId() == 'node' && $entity->getType() == 'article') {
+    $material = $entity->get('field_ingredients')->value;
+    $ingredients = explode(' ', $material);
+    echo $ingredients;
+  }
+  foreach ($ingredients as $ingredient) {
```

Comment:
What happens if the site administrator creates a Basic Page (or any other content type other than an Article)?

Reply:
Do you mean I should place this piece of code inside the "if" statement, because this way it is not only applies for Articles but for every type?